tests/CLI/Command/Convert/DMTest.php

```diff
@@ -119,7 +116,6 @@ public function testExecuteWithEllipsoidOption()
             '--ellipsoid' => 'AUSTRALIAN_NATIONAL',
         ));
 
-        $this->assertTrue(is_string($this->commandTester->getDisplay()));
-        $this->assertMatchesRegularExpression('/40 26\.7717N, -79 56\.93172W/', $this->commandTester->getDisplay());
+        $this->assertEquals(0, $this->commandTester->getStatusCode());
```

Comment:
How come we are not testing the `display` anymore?

Reply:
The display is always a string, I think is useless to test it. 
I also removed tests about the result of commands, because the math calculation is already tested in their own test class, and I think it's a bad practice to test several times.

Reply:
Friendly ping @Nyholm :)